boards/arm/sam4s_xplained/Makefile

```diff
@@ -0,0 +1,9 @@
+# Makefile - Atmel SAM4S Xplained Board
+#
+# Copyright (c) 2017 Justin Watson
+# SPDX-License-Identifier: Apache-2.0
+#
+
+ccflags-y += -I$(srctree)/include/drivers
```

Comment:
Ok, I will remove. This means that the SAM3X also needs them removed because I looked at it originally when doing this work.